linera-service/src/linera/main.rs

```diff
@@ -380,12 +380,15 @@ impl Runnable for Job {
                 }
 
                 let genesis_config_hash = context.wallet().genesis_config().hash();
-                match node.get_genesis_config_hash().await {
-                    Ok(hash) if hash == genesis_config_hash => {}
-                    Ok(hash) => error!(
-                        "Validator's genesis config hash {} does not match our own: {}.",
-                        hash, genesis_config_hash
-                    ),
+                match node.get_network_description().await {
+                    Ok(description) => {
+                        if description.genesis_config_hash != genesis_config_hash {
+                            error!(
+                                "Validator's genesis config hash {} does not match our own: {}.",
+                                description.genesis_config_hash, genesis_config_hash
+                            );
+                        }
+                    }
```

Comment:
Here it might be useful to also print both network names. (Also below.)

Reply:
Mmm I'm going to send another PR for this.